src/packages/default-configuration/signer.ini

```diff
@@ -23,3 +23,6 @@ key-length=2048
 ; Certificate signing request signature digest algorithm,
 ; possible values: SHA-256, SHA-384, SHA-512
 csr-signature-digest-algorithm=SHA-256
+
+; Optional comma separated list of HSM slot indexes
+;hsm-slot-indexes=0,7,12
```

Comment:
There is no need to put this parameter to signer.ini. End users should add that to local.ini manually.